learningequality/studio
# Fix channel thumbnail upload

## Description

Fixes the behaviour for saving thumbnail_encoding data to the server. As we are using a patch update for this, it does not get passed through the 'toJSON' method of the Channel Model on the frontend, and so is not getting JSON.stringified. Does away with the need for this by storing this data on the backend as a postgres JSONField.

#### Before/After Screenshots (if applicable)

Before:
![channelthumbnailbefore gif](https://user-images.githubusercontent.com/1680573/50315394-6f7b3980-0467-11e9-82bb-cdc7a61892f4.gif)

After:
![channelthumbnailafter](https://user-images.githubusercontent.com/1680573/50315401-74d88400-0467-11e9-8df0-3d08d3e534d2.gif)


## Steps to Test

- Open the channel details tab
- Click 'Edit Details'
- Upload a new thumbnail
- Save your changes
- See no flappy bird/async error
## Checklist


## Comments

Need to test the last checklist item before merging. This involves individual operations on each channel, so may be very slow.
